modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -382,8 +382,17 @@ public boolean stopReplica(ReplicationGroupId replicaGrpId) throws NodeStoppingE
      * @param replicaGrpId Replication group id.
      * @return True if the replica is found and closed, false otherwise.
      */
+    // TODO: IGNITE-19494 We need to correctly stop the replica
```

Comment:
The ticket that is mentioned here has a terrible description, sorry

Reply:
I tried to paraphrase, but I don’t know what else to add, do you advise something?

Reply:
Instead of waiting we should complete the future exceptionally in order to interrupt replica await process. Semantically, only if it's not completed already, but who cares completeExceptionally will do the trick. We may introduce ReplicaStoppingException or similar and verify that such kind of exception is properly handled in `ReplicaService#sendToReplica`.
@vldpyatkov @sergeyuttsel Guys you've touched replica await code recently, do you have any objections?

Reply:
I have already created IGNITE-19494 in which it will be possible to implement any behavior, I propose this within wdyt?

Reply:
I have no objections. I agree that we should complete the future exceptionally in order to interrupt replica await process in ReplicaService#sendToReplica.

Reply:
Yep. We(TX teem) will do this.